lib/rspecq/reporter.rb

```diff
@@ -83,6 +83,14 @@ def summary(failures, errors, flaky_jobs, duration)
       end
 
       summary = ""
+      if @queue.suite_failed?
+        summary << "\n\n"
+        summary << "**Fail fast was enabled for this run**(threshold=#{@queue.fail_fast})"
```

Comment:
Since this is the output that's going to be printed to the user, let's try to make it more readable. How about:

> The limit of #{@queue.fail_fast} failures has been reached in this build! Aborting...